asset/file_test.go

```
@@ -0,0 +1,311 @@
+			},
+		},
+		{
+			name:  "file object is empyy",
+			files: nil,
+			want:  nil,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+			flatten := tt.files.FlattenChildren()
```

golangci-lint (1.64.8)

loopclosure: loop variable tt captured by func literal

(govet)